exodus_gw/settings.py

```diff
@@ -62,6 +69,15 @@ def get_settings() -> Settings:
         ]
     )
 
+    for logger in config["loglevels"] if "loglevels" in config else []:
+        if "loggers" not in settings.log_config:
+            settings.log_config.update({"loggers": {}})
```

Comment:
There's a built-in method for this to handle it more concisely:

```suggestion
        settings.log_config.setdefault("loggers", {})
```